standalone/src/models/order/tracking-data/presets/index.ts

```diff
@@ -0,0 +1,3 @@
+const presets = {};
```

Comment:
We should export the three types of presets for consistency:
```
export const restPresets = {};
export const graphqlPresets = {};
export const compatPresets = {};
```

This makes sure if we need to add a preset in the future, the structure is already there for whoever needs to do it.